### PR TITLE
itemsQuestionModel added

### PR DESCRIPTION
**Pull request by jamesrea83**
Added itemsQuestionComponentModel #1760

### CODE REVIEW

**Comment by oliverfoster** [comment]
Could you rename to `ItemsQuestionModel`?

File: src/core/js/models/itemsQuestionComponentModel.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+			this.get('_items').each(function(item) { item.reset(); });
+
+			QuestionModel.prototype.reset.call(this, type, force);
+		},
+
+		resetActiveItems: function() {
+			this.get('_items').each(function(item) { item.toggleActive(false); });
+		},
+
+        setActiveItem: function(index) {
```
Comment (oliverfoster) [inline src/core/js/models/itemsQuestionComponentModel.js]: indentation isn't consistent

**Comment by chetan-hajare** [comment]
@oliverfoster Just want to know that are we planning to change the indentation from 4 spaces to 8 spaces?

**Comment by lc-thomasberger** [comment]
Should be 4

**Comment by moloko** [comment]
> are we planning to change the indentation from 4 spaces to 8 spaces?

@chetan-hajare not while I live and breathe... I compromised enough switching away from tabs!

**Comment by moloko** [comment]
Actually I note that this is using tabs for indentation so +100 ;-)

**Comment by jamesrea83** [comment]
Closing, will create a new PR.

**Pull request closed without merging**